translate/automl/src/main/java/com/google/cloud/translate/automl/GetDataset.java

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright 2019 LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.cloud.translate.automl;
+
+// [START automl_translate_get_dataset]
+import com.google.cloud.automl.v1.AutoMlClient;
+import com.google.cloud.automl.v1.Dataset;
+import com.google.cloud.automl.v1.DatasetName;
+
+import java.io.IOException;
+
+class GetDataset {
+
+  // Get a dataset
+  static void getDataset(String projectId, String datasetId) {
+    // String projectId = "YOUR_PROJECT_ID";
+    // String datasetId = "YOUR_DATASET_ID";
+
+    // Initialize client that will be used to send requests. This client only needs to be created
+    // once, and can be reused for multiple requests. After completing all of your requests, call
+    // the "close" method on the client to safely clean up any remaining background resources.
+    try (AutoMlClient client = AutoMlClient.create()) {
+      // Get the complete path of the dataset.
+      DatasetName datasetFullId = DatasetName.of(projectId, "us-central1", datasetId);
+      Dataset dataset = client.getDataset(datasetFullId);
+
+      // Display the dataset information
+      System.out.format("Dataset name: %s\n", dataset.getName());
+      System.out.format(
+          "Dataset id: %s\n",
+          dataset.getName().split("/")[dataset.getName().split("/").length - 1]);
```

Comment:
ok, but could you clean this up a bit and teach something here?  Why do I see the same code in two places?

Reply:
The `create`, `get`, `list` for both datasets and models are all very similar.